celer/group_fast.pyx

```diff
@@ -237,24 +237,24 @@ cpdef celer_grp(
         tmp = 1. / n_samples
         fscal(&n_samples, &tmp, &theta[0], &inc)
 
-        scal = dnorm_grp(
+        dnorm = dnorm_grp(
             is_sparse, theta, grp_ptr, grp_indices, X, X_data, X_indices,
             X_indptr, X_mean, weights, n_groups, dummy_C, center)
 
-        if scal > alpha:
-            tmp = alpha / scal
+        if dnorm > alpha:
+            tmp = alpha / dnorm
```

Comment:
can you also rename tmp to `theta_scaling` ? (name open to discussion :) )

Reply:
I agree with the name